feature-staking-impl/src/main/java/jp/co/soramitsu/staking/impl/di/StakingUpdatersModule.kt

```diff
@@ -311,7 +311,7 @@ class StakingUpdatersModule {
             accountValidatorPrefsUpdater,
             accountNominationsUpdater,
             rewardDestinationUpdater,
-            historyDepthUpdater,
+//            historyDepthUpdater,
```

Comment:
do we need to keep this comment?

Reply:
yeah, let's keep it for some time. When I will be sure that we don't need this updater anymore, I will remove it.